apps/golden-sample-app/src/app/theme-switcher/theme-service.ts

```diff
@@ -0,0 +1,15 @@
+import { Injectable } from '@angular/core';
+
+@Injectable()
+export class ThemeService {
+  public static THEMES = [
+    'theme-default',
+    'theme-premium',
+  ];
+
+  public static setTheme(theme: any):void {
```

Comment:
Do you need these to be `static`? How about following the documentation in https://designsystem.backbase.com/theme/multi-theming-on-web#2-implementing-the-theme-manager? If you want to it a different way, you may need to update the documentation also.

Reply:
updated